example/src/testing/Tests/CipherTests/PublicCipherTests.ts

```diff
@@ -40,19 +40,19 @@ function testEncryptDecrypt(publicKey: any, privateKey: any) {
 //   }
 // }
 
-export function registerPublicCipherTests() {
-  // We need to monkey patch sscrypto to use all the crypto functions from quick-crypto
-  it('sscrypto basic test', async () => {
-    try {
-      const clearText = 'This is clear text';
-      const privateKey = await PrivateKey.generate(1024);
-      const encrypted = privateKey.encrypt(Buffer.from(clearText) as any);
-      const decrypted = privateKey.decrypt(encrypted);
-      chai.expect(decrypted.toString('utf-8')).to.equal(clearText);
-    } catch (e) {
-      chai.assert.fail();
-    }
-  });
+describe('publicCipher', () => {
```

Comment:
Why do we comment it out?

Reply:
the two sscrypto tests (here and somewhere else) never got the monkey patch that is referenced in the comment.  So I commented them out for now.